buildpacks/nodejs-npm-install/src/bin/npm_runtime_config.rs

```diff
@@ -0,0 +1,52 @@
+use commons as _;
+use fastrand::alphanumeric;
+use fun_run::{CmdError, CommandWithName, NamedOutput};
+use heroku_nodejs_utils as _;
+use indoc as _;
+use libcnb as _;
+#[cfg(test)]
+use libcnb_test as _;
+use serde as _;
+#[cfg(test)]
+use serde_json as _;
+use std::env::temp_dir;
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::process::Command;
+#[cfg(test)]
+use test_support as _;
+
+fn main() {
+    if let Err(e) = disable_update_notifier() {
+        println!("{e}");
+    }
+
+    if let Err(e) = set_temp_cache_dir() {
+        println!("{e}");
+    }
+}
+
+fn disable_update_notifier() -> Result<NamedOutput, CmdError> {
+    Command::new("npm")
+        .args(["config", "set", "update-notifier", "false"])
+        .named_output()
+}
+
+fn set_temp_cache_dir() -> Result<NamedOutput, CmdError> {
+    Command::new("npm")
+        .args(["config", "set", "cache"])
+        .arg(create_temp_dir_name("npm.XXXXXXX"))
+        .named_output()
+}
+
+fn create_temp_dir_name(template: &str) -> PathBuf {
```

Comment:
Do we need a randomized name here? It sounds weird to recreate a new one every reboot. It seems like a static name would be fine, and that'd probably shrink the size of this binary on the application image, too.

Reply:
this was driven more by my own expectations around temp dir creation which usually involves a randomly generated file/folder name. i can drop it if you don't think name collisions will ever be a concern.

Reply:
switched to using `/tmp/npm_cache` as the name and dropped the randomization